### PR TITLE
Adding jaguares pool to registry.json

### PR DESCRIPTION
**Pull request by Danvegamo**
### Commitment

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist

I have:
- [x] Added a telegram handle of an **individual** and not that of a group or channel.
- [x] Added a social media account with activity.
- [x] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [x] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [x] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

### CODE REVIEW

**Comment by Danvegamo** [comment]
I'm waiting for approval

File: registry.json
Comment (Bez625) [inline registry.json:5487]: Please remove the @

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json
Comment (Bez625) [inline registry.json:5487]: Please can you add some social media activity? The instagram account seems empty...

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by Bez625** [comment]
Please address the comments or I will look to close in the coming weeks

**Comment by Bez625** [comment]
Closing as there have been no updates since changes were requested. Feel free to raise another PR if you still want to join

**Pull request closed without merging**
